src/style.css

```diff
@@ -0,0 +1,21 @@
+body {
+  margin: 0;
+}
+
+.block_1 {
+  background: red;
+  height: 100px;
+  width: auto;
```

Comment:
Block elements has 100% of the width of the parent container. Set width of 100% is redundant